plugins/woocommerce-admin/client/two-column-tasks/headers/purchase.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import TimerImage from './timer.svg';
+import { WC_ASSET_URL } from '../../utils/admin-settings';
+
+const PurchaseHeader = ( { task, goToTask } ) => {
+	return (
+		<div className="woocommerce-task-header__contents-container">
+			<img
+				alt={ __( 'Purchase illustration', 'woocommerce' ) }
+				src={
+					WC_ASSET_URL + 'images/task_list/purchase-illustration.png'
+				}
+				className="svg-background"
+			/>
+			<div className="woocommerce-task-header__contents">
+				<h1>{ task.title }</h1>
+				<p>
+					{ __(
+						'Good choice! You chose to add amazing new features to your store. Continue to checkout to complete your purchase.',
+						'woocommerce'
+					) }
+				</p>
+				<Button
+					isSecondary={ task.isComplete }
+					isPrimary={ ! task.isComplete }
+					onClick={ goToTask }
```

Comment:
See the custom purchase task item we have here: https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/tasks/fills/purchase.js

I know for other headers like WooCommerce Payments we copied that custom logic to the header also instead of using `goToTask` -> https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce-admin/client/two-column-tasks/headers/woocommerce-payments.js#L86